src/utils/uint256.ts

```diff
@@ -10,14 +10,34 @@ export { UINT_128_MAX, UINT_256_MAX };
 /**
  * Convert Uint256 to bigint
  * Legacy support Export
+ * @param uint256 Uint256 value to convert to bigint
+ * @returns {bigint} BigInt representation of the input Uint25
+ * @example
+ * ```typescript
+ * const uint256Value: Uint256 = { low: 1234567890, high: 1 };
+ * const result = uint256ToBN(uint256Value);
```

Comment:
All these functions are in the namespace `uint256`. 
Users have to be able to perform these tests.
Here, without the namespace, it's not possible.
I don't understand how you have succeeded to test these examples!